src/molecule_podman/playbooks/create.yml

```diff
@@ -5,10 +5,12 @@
   gather_facts: false
   no_log: "{{ molecule_no_log }}"
   become: "{{ not (item.rootless|default(true)) }}"
+  vars:
+    podman_exec: "{{ lookup('env','MOLECULE_PODMAN_EXECUTABLE')|default('podman',true) }}"
   tasks:
     - name: Log into a container registry
       command: >
-        podman login
+        {{ podman_exec }} login
```

Comment:
The same sentiment goes here: use an absolute path.

Reply:
I've added a task that determines the path to the executable and uses that for login and build